content/community/tools-data-fetching.md

```diff
@@ -0,0 +1,12 @@
+---
+id: data-fetching
+title: Data Fetching
+layout: community
+permalink: community/data-fetching.html
+---
+
+* **[Apollo](http://dev.apollodata.com/react/):** Easy to set up and use GraphQL client.
+* **[Axios](https://github.com/mzabriskie/axios):** Promise based HTTP client for the browser and node.js.
```

Comment:
I'm surprised that URLs like these actually get resolved to GitHub links, but they seem to 😮